src/middlewared/middlewared/plugins/alert.py

```diff
@@ -399,6 +400,10 @@ def restore(self, id):
                             pass
                         else:
                             raise
+                except ReserveFDException:
+                    self.logger.debug('Failed to reserve a privileged port')
+                    pass
```

Comment:
I will be picky here and say `pass` has become a no-op/useless

Reply:
:)